src/treeview/DbConnectionUpdater.ts

```diff
@@ -1,27 +1,60 @@
 import * as vscode from "vscode";
 import { DatabaseProcessor } from "../db/DatabaseProcessor";
-import { IConfig } from "../view/app/model";
+import { ConnectionStatus, IConfig } from "../view/app/model";
 import { IRefreshCallback, RefreshWithoutCallback } from "./IRefreshCallback";
 
 export class DbConnectionUpdater {
     constructor (){}
+    private locked: boolean = false;
 
     public async updateConnectionStatuses (context: vscode.ExtensionContext) {

```

Comment:
This method should be removed as it becomes obsolete. Also remove `RefreshWithoutCallback `class in IRefreshCallback.ts